parts/kubernetesmastercustomscript.sh

```diff
@@ -130,6 +130,23 @@ function ensureKubelet() {
 function extractKubectl(){
     systemctl enable kubectl-extract
     systemctl restart kubectl-extract
+    kubectlExtracted=1
```

Comment:
Thanks for your feedback! I think I can move the systemd unit after the kubelet.service. If the kubelet service is started correctly, then the kubectl extraction should be fine. Does it make sense?